release/pipline/dags/istio_common_dag.py

```diff
@@ -174,7 +177,15 @@ def GenerateTestArgs(**kwargs):
     git config --global user.email "testrunner@istio.io"
     git clone {{ settings.MFEST_URL }} green-builds || exit 2
     cd green-builds
-    git checkout {{ settings.MFEST_COMMIT }} || exit 3
+    SHA=`grep {{ settings.MFEST_FILE }} | cut -f 6 -d \\"` || exit 3
```

Comment:
To me it looks like only 1 parameter is passed to grep.  I'm thinking there needs to be something like:
grep {{ settings.GITHUB_ORG }}/{{ settings.GITHUB_REPO }} {{ settings.MFEST_FILE }}

Reply:
Done